### DISCUSSION TITLE
Stakeholder needs

### DISCUSSION THREAD

**Discussion by capsulecorplab** [ODU-CGA-CubeSat/sealion-mission-architecture#20] [Ideas]
As per the first principle in systems engineering, this discussion is for identifying stakeholder needs.

**Comment by capsulecorplab** [comment]
Draft for stakeholder need statement: 
_To verify deployment of composite structure on a CubeSat_

Note: need to specify "composite structure"

**Reply by capsulecorplab** [reply]
Would appending _as a successor to the ThinSat concept_ be superfluous?

**Comment by capsulecorplab** [comment]
Stakeholder needs will likely just be the mission objectives, as per presentation in the June 24th meeting

**Reply by paclark5** [reply]
1.1 Establish UHF communication link with Virginia ground station
1.2 Establish S-Band communication link with MC3 ground station
2.1 On-orbit validation of impedance probe
2.2 On-orbit validation of VIR-S
2.3 On-orbit validation DeCS experiment with accelerometer data
3.1 Qualify newly developed antenna
3.2 Qualify very-low Earth orbit bus
3.3 DeCS in-orbit behavioral performance via accelerometer and temperature sensor data

**Reply by capsulecorplab** [reply]
What is VIR-S?

**Reply by paclark5** [reply]
V-Infrared Sensor
![virs](https://user-images.githubusercontent.com/84911927/124510656-4bbc3c80-dda2-11eb-8e9c-2241010bb6eb.JPG)

**Comment by AstroNiashree** [comment]
Components Lists with Pro and Cons

**Reply by AstroNiashree** [reply]
Sun Sensors
1.	Solar MEMS Technologies S.L.
1.	nanoSSOC-A60 - Analog Sun Sensor for Nanosatellites
2.	SSOC-A60 - Analog Sun Sensor on a Chip
3.	nanoSSOC-A60 - Analog Sun Sensor for Nanosatellites
2.	The FSS100 – Nano Fine Sun Sensor by Tensor Tech
3.	The NCSS-SA05 and NFSS-411 by NewSpace Systems
4.	The SS200 by Hyperion Technologies
GPS Receivers
1.	WARPSPACE GPS Receiver
2.	GNSS Receiver Module (GPSRM 1) Kit by Pumpkin
3.	GPSRM 1 GPS Receiver Module
4.	DLR Phoenix Phoenix GPS receiver
5.	Accord Software & Systems Private Limited ACC-GPS-NANO-ACTIVE

**Reply by AstroNiashree** [reply]
Link for Batteries
[https://www.eaglepicher.com/
](url)

[https://www.ultralifecorporation.com/ECommerce/product/uhr-xr34610-cell/3v-li-cfx-mno2-hybrid-non-rechargeable-d-cell/uhr-xr34610-s/16ah-33v-cell-only
](url)

**Reply by AstroNiashree** [reply]
Follow the Physical Dimensions (Weight, Height, diameter etc) for the battery.

**Reply by AstroNiashree** [reply]
Regulator Board (battery Bus) PC104 @nanomind

**Reply by AstroNiashree** [reply]
Link for Sun sensors [https://satsearch.co/products/bradford-coarse-sun-sensor](url)
[https://blog.satsearch.co/2020-02-12-sun-sensors-an-overview-of-systems-available-on-the-global-marketplace-for-space](url)

**Reply by BhagatjiJimesh** [reply]
Here are the specific of **On-orbit validation DeCS experiment**:
1) Validation of deployable mechanism for a composite boom. 
2) Qualification of deployable composite boom dynamics.
